app/models/grda_warehouse/cohorts/document_exports/cohort_excel_export.rb

```diff
@@ -53,8 +55,10 @@ def perform
 
         write_tmp_file(
           renderer.render(
+            template: 'cohorts/export',
             action: :show,
-            format: :xlsx,
```

Comment:
Not sure that all of these changes are needed. Going broad since it's hard to reproduce